### ISSUE TITLE
Error code 52000 and 52100

### ISSUE THREAD

**Issue by bigbeau** [barronwaffles/dwc_network_server_emulator#259]
so I apparently have no idea what I'm doing... I'm using a DS with a DS(i) action replay on Pokemon White and Mario kart DS, and no matter what I do it will not connect to a server. Can someone help? error code 52000 isn't even on the troubleshooting page...

**Comment by Saiyaken-PHOENIX** [comment]
may be an issue with your wireless router

**Comment by omarrrio** [comment]
I managed to bypass the 52000 error, you need to follow these instructions:
The first thing you'll need to do is find out the range of IP addresses you should be using. To do this:

```
Click the "Start" button in the lower left-hand corner of your computer.
Click "Run."
Type "cmd" into the box and press Enter.
Type "IPCONFIG/ALL" and press Enter. The IP address of the PC will be displayed.
The IP address you are going to assign to the Nintendo DS must be 10 digits higher than the PC's. (e.g., 192.168.1.100 becomes 192.168.1.110).
Enter in the exact same values for your Subnet Mask, Default Gateway, Primary DN S and Secondary DNS as you see listed on your computer screen for IPCONFIG/ALL.
```

Although i am now stuck at 52100.

**Comment by bigbeau** [comment]
Yeah, IPCONFIG is actually how I determined what IP to use for the DS in the first place. After I made a port for it on my router, error 52100 is all that popped up from then on out. Whats funny is, I can access the server on my homebrew'd Wii just fine. Also, my cat decided it would be nice to break my action replay (like tear the case and pull the card holder off the PCB. I don't even know how it got it open), so until I can get a new one or a DSTWOplus I unfortunately can't really try anything...
